target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -1131,8 +1252,11 @@ TARGET_DEVICES += CPA-ZNTE60T
 
 define Device/MAGIC
   $(Device/bcm63xx)
+  DEVICE_VENDOR := Telsey
+  DEVICE_MODEL := Alice W-Gate
+  DEVICE_ALT0_VENDOR := Telsey
+  DEVICE_ALT0_MODEL := MAGIC
```

Comment:
Alice is the vendor of the W-Gate, while Telsey is the Vendor (OEM) of the MAGIC.

I have no idea whether this is the black or the white W-Gate, and if there even is a difference internally.

Reply:
Ok, that's correct then?
```
DEVICE_VENDOR := Alice
DEVICE_MODEL := W-Gate
DEVICE_ALT0_VENDOR := Telsey
DEVICE_ALT0_MODEL := MAGIC
```

Reply:
Looks good